sources/niCURL/src/_build.ham

```diff
@@ -12,6 +12,12 @@ if ! ( $(USE_SYSTEM_LIBCURL) = 1 ) {
   SubDirHdrs $(SUBDIR) libCURL include ;
 }
 
+if ( $(EMSCRIPTEN) = 1 ) {
+  LINKFLAGS ?= $(LINKFLAGS) ;
+  LINKFLAGS += -s EXPORTED_FUNCTIONS=_main,_malloc,_free,_FetchOverride_Success,_FetchOverride_Error,_FetchOverride_Progress
+    ;
+}
+
```

Comment:
I really dont know how to do this...I tried a lot of different ways. I just want to se extra linking flags ONLY FOR niCURL.
I tried LINKFLAGS on niCURL, LINKFLAGS on 
(<) : and also creating _rules.ham in the niCURL folder and overrite the flags there.

As I have it this PR it works if I run ham all inside the niCURL directory, the problem I have is that when I run ham all in the root of niLang, other projects like niUI are trying to link _FetchOverride_Error also.

Reply:
I think what you want is to add this in _rules_sdks.ham at https://github.com/prenaux/niLang/blob/3a64369f19c604b8193c745fe3159680fbfb81db/sources/_rules_sdks.ham#L226